### PR TITLE
refactor(rxUtcOffsets): rx-prefixes update UtcOffsets

### PR DESCRIPTION
**Pull request by cathysiller**
JIRA: https://jira.rax.io/browse/FRMW-1164

### LGTMs
- [x] Dev LGTM
- [x] Dev+Vidyo LGTM

### CODE REVIEW

File: src/elements/Forms/scripts/rxTimePicker.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ angular.module('encore.ui.elements')
         link: function (scope, element, attrs, ngModelCtrl) {
             var pickerUtil = rxTimePickerUtil;
 
-            scope.availableUtcOffsets = UtcOffsets;
+            scope.availablerxUtcOffsets = rxUtcOffsets;
```
Comment (CITguy) [inline src/elements/Forms/scripts/rxTimePicker.js]: Please revert `scope.availablerxUtcOffsets` back to `scope.availableUtcOffsets` (left hand side only).

File: src/elements/Forms/templates/rxTimePicker.html [hunk 2]
```diff
@@ -60,7 +60,7 @@
                   ng-model="offset">
 
                   <option
-                    ng-repeat="utcOffset in availableUtcOffsets"
+                    ng-repeat="utcOffset in availablerxUtcOffsets"
```
Comment (CITguy) [inline src/elements/Forms/templates/rxTimePicker.html]: Please revert `availablerxUtcOffsets` back to `availableUtcOffsets`.

File: src/elements/Forms/scripts/rxTimePicker.js (hunk 1 above)
Comment (cathysiller) [inline src/elements/Forms/scripts/rxTimePicker.js]: reverted (left side only)

File: src/elements/Forms/templates/rxTimePicker.html (hunk 2 above)
Comment (cathysiller) [inline src/elements/Forms/templates/rxTimePicker.html]: reverted

**Comment by lentusignavus** [comment]
Should the files be renamed as well? @cathysiller @CITguy

**Comment by CITguy** [comment]
Ah, good catch @lentusignavus. Yes, they should definitely be renamed.

File: src/utilities/rxUtcOffsets/docs/UtcOffsets.meta.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (CITguy) [inline src/utilities/rxUtcOffsets/docs/UtcOffsets.meta.json]: Please rename file.

File: src/utilities/rxUtcOffsets/scripts/UtcOffsets.js [hunk 4]
```diff
@@ -1,15 +1,15 @@
 angular.module('encore.ui.utilities')
```
Comment (CITguy) [inline src/utilities/rxUtcOffsets/scripts/UtcOffsets.js]: Please rename file.

File: src/utilities/rxUtcOffsets/docs/UtcOffsets.meta.json (hunk 3 above)
Comment (cathysiller) [inline src/utilities/rxUtcOffsets/docs/UtcOffsets.meta.json]: renamed -- thanks for catching @lentusignavus  !!

File: src/utilities/rxUtcOffsets/scripts/UtcOffsets.js (hunk 4 above)
Comment (cathysiller) [inline src/utilities/rxUtcOffsets/scripts/UtcOffsets.js]: renamed !!

**Comment by lentusignavus** [comment]
LGTM

**Pull request merged**